charts/README.md

```diff
@@ -0,0 +1,109 @@
+# feathr-online
```

Comment:
Please do not move this file. As there will be more charts like feathr-sandbox, feathr-registry under directory charts.

Reply:
Ok sure!